balancer-js/examples/joinGeneralisedComposableStable.ts

```diff
@@ -44,26 +46,26 @@ const balancer = new BalancerSDK({
       where: {
         address: {
           in: [
-            "0xa13a9247ea42d743238089903570127dda72fe44",
-            "0xae37d54ae477268b9997d4161b96b8200755935c",
-          ]
-        }
+            '0xa13a9247ea42d743238089903570127dda72fe44',
+            '0xae37d54ae477268b9997d4161b96b8200755935c',
```

Comment:
@gmbronco - not sure why, but it feels that your prettier is not active for the examples folder. Would be nice to double check, so I don't trigger several unexpected changes after changing a single line of code 😕